src/model/line_webhook.rs

```diff
@@ -0,0 +1,143 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Webhook {
+    pub destination: String,
+    pub events: Vec<EventType>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(tag = "type")]
+#[serde(rename_all = "camelCase")]
+pub enum EventType {
+    Message(MessageEvent),
+    Follow(FollowEvent),
+    Unfollow(UnfollowEvent),
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct MessageEvent {
+    #[serde(rename = "replyToken")]
+    pub reply_token: String,
+    pub mode: String,
+    pub timestamp: i64,
+    pub source: Option<SourceType>,
+    #[serde(rename = "webhookEventId")]
+    pub webhook_event_id: String,
+    #[serde(rename = "deliveryContext")]
+    pub delivery_context: DeliveryContext,
+    pub message: MessageType,
+}
```

Comment:
構造体などの定義は参照先をコメントで書いておくと便利です！

```suggestion
// https://developers.line.biz/ja/docs/basics/channel-access-token/
#[derive(Debug, Serialize, Deserialize)]
pub struct MessageEvent {
    #[serde(rename = "replyToken")]
    pub reply_token: String,
    pub mode: String,
    pub timestamp: i64,
    pub source: Option<SourceType>,
    #[serde(rename = "webhookEventId")]
    pub webhook_event_id: String,
    #[serde(rename = "deliveryContext")]
    pub delivery_context: DeliveryContext,
    pub message: MessageType,
}
```